styles/card.scss

```diff
@@ -0,0 +1,24 @@
+.c-card {
+	color: var(--color-white);
+	max-width: 320px;
+}
+
+.c-card__info {
+	margin-top: 20px;
+
+	> p {
+		margin-top: 8px;
+		color: var(--color-suva-gray);
+		font-weight: bold;
+		font-size: 0.875rem;
+	}
+}
+.c-card__info-year {
+	font-size: 0.875rem;
+	float: right;
+	padding: 4px 16px;
+	border: 1px solid var(--color-emperor);
+	border-radius: 3px;
+	box-sizing: border-box;
+	display: inline-block;
```

Comment:
you don't need "display: inline-block" because you have "float: right", this element is getting "display: block" by default